docs/getting-started.md

```diff
@@ -0,0 +1,181 @@
+﻿# Kubevirtci
+
+The purpose of Kubevirtci is to create pre-provisioned Kubernetes/OpenShift clusters as container images,\
```

Comment:
why is the trailing backslash? We should let the HTML rendering decide where to break lines.

Reply:
the lines get very long without it, i can remove later and you can see and decide